components/ipfs/ipfs_utils.cc

```diff
@@ -203,52 +203,54 @@ bool TranslateIPFSURI(const GURL& url,
   if (!url.SchemeIs(kIPFSScheme) && !url.SchemeIs(kIPNSScheme)) {
     return false;
   }
+  if (!url.host().empty())
+    return false;
 
-  std::string path = url.path();
+  // ipfs: or ipfs://
```

Comment:
@spylogsster wuick question:

- Does this mean whichever variant is used, that will stay in address bar?
   Or is one of them "canonical" one that we always normalize to?

Reply:
Whichever variant is used, that will stay in address bar. This is only for explanation why we check for substr(0,2) == "//", I agree it's not as clear as it could be ;)